### PR TITLE
Feature/fix mongo issues

### PR DESCRIPTION
**Pull request by robinandeer**
This time it would be good with a proper code review @moonso ^^

### CODE REVIEW

File: scout/ext/backend/load_mongo.py [hunk 1]
```diff
@@ -135,27 +127,27 @@ def load_mongo_db(scout_configs, config_file=None, family_type='cmms',
...
-  
+  ped_individuals = {individual.individual_id: individual.display_name
+                     for individual in case.individuals}
+
   # Check which individuals that exists in the vcf file:
   individuals = []
-  for individual in ped_individuals:
-    if individual in variant_parser.individuals:
-      individuals.append(individual)
+  # loop over keys (internal ids)
```
Comment (moonso) [inline scout/ext/backend/load_mongo.py]: We need to have the same ids in this list as the ones in the vcf

**Comment by moonso** [comment]
Fixed the stuff we talked about, please have a look.

**Comment by moonso** [comment]
So what do you think @robinandeer , is this ready for merge?

**Comment by robinandeer** [comment]
I have tested locally with reloaded data and it seems to work fine!

Deploy :shipit:

**Comment by robinandeer** [comment]
I will merge then ^^

**Comment by moonso** [comment]
Nice!

**Pull request merged**